addons/jenkins/1.x/jenkins.yaml

```
@@ -6,7 +6,7 @@ metadata:
   labels:
     kubeaddons.mesosphere.io/name: jenkins
   annotations:
-    catalog.kubeaddons.mesosphere.io/addon-revision: "1.9.4-1"
+    catalog.kubeaddons.mesosphere.io/addon-revision: "1.9.4-2"
```

```suggestion
    catalog.kubeaddons.mesosphere.io/addon-revision: "2.5.4-1"
```
This repo should follow the guidelines in https://github.com/mesosphere/kubernetes-base-addons/blob/master/CONTRIBUTING.md#addon-revisions IMHO.